release-notes/8.0/install.md

```diff
@@ -55,7 +55,7 @@ Download links are provided for each of the distributions at:
 - [Microsoft .NET website](https://dotnet.microsoft.com/download/dotnet/8.0)
 - [.NET 8 release notes](https://github.com/dotnet/core/blob/main/release-notes/8.0/preview/8.0.0-preview.7.md)
 
-[Container images](https://hub.docker.com/r/microsoft/dotnet/) are provided for Windows (Nano Server, and Server Core) and Linux (Alpine, Debian, and Ubuntu).
+[Container images](https://hub.docker.com/r/microsoft/dotnet/) are provided for Linux (Alpine, Debian, Ubuntu, Azure Linux, and distroless variants of Ubuntu and Azure Linux) and Windows (Nano Server and Server Core).
```

Comment:
This is pointing to DH but the main README points to MAR. We should be consistent where we direct people.

Reply:
I agree. I would like to point everything to MAR, but MAR doesn't have product family repos like https://hub.docker.com/r/microsoft/dotnet/. Is the SDK repo on MAR an acceptable replacement or should we send all the links to DH? Or what about linking straight to https://github.com/dotnet/dotnet-docker?

@richlander and @MichaelSimons for input

Reply:
Both are bad?

I'm very torn. Ideally, we primarily reference MAR, but we also need to help people where to look on DH because a lot (perhaps most) of people look there. It's challenging.

Reply:
I changed all of the links to DH because
1. We have the product family repo there
2. We have greater control over the tag discovery/organization situation in our DockerHub readmes
3. Ultimately customers are pulling from mcr.microsoft.com either way.